### PR TITLE
Update to Aspire preview7 [WIP]

### PR DESCRIPTION
**Pull request by adityamandaleeka**
TODO: Undo feed changes when shipped.

### CODE REVIEW

**Review by danmoseley — APPROVED** [review]
Everything works fine?

**Comment by danmoseley** [comment]
Can we do the Azure one as well?

**Comment by adityamandaleeka** [comment]
Closing in favor of #331

File: Directory.Packages.props [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <AspnetVersion>8.0.3</AspnetVersion>
     <MicrosoftExtensionsVersion>8.3.0</MicrosoftExtensionsVersion>
     <EfVersion>8.0.3</EfVersion>
-    <AspireVersion>8.0.0-preview.5.24201.12</AspireVersion>
+    <AspireVersion>8.0.0-preview.6.24214.1</AspireVersion>
```
Comment (adityamandaleeka) [inline Directory.Packages.props:8]:
```suggestion
    <AspireVersion>8.0.0-preview.7.24251.11</AspireVersion>
```

**Pull request closed without merging**